garage/misc/wrappers.py

```diff
@@ -0,0 +1,88 @@
+"""Gym env wrappers"""
+import cv2
+import gym
+import numpy as np
+
+from collections import deque
+from gym.spaces import Box
+
+
+class RepeatActionWrapper(gym.core.Wrapper):
```

Comment:
I think you should be inheriting from `gym.Wrapper` rather than `gym.core.Wrapper`